config/application.rb

```diff
@@ -108,6 +108,9 @@ class Application < Rails::Application
     # The directory in which repositories should be cached.
     config.samson.cached_repos_dir = Rails.root.join("cached_repos")
 
+    # Choose the remote repository for this Samson implementation.  Options are github or gitlab
+    config.samson.remote_repository = ENV["REMOTE_REPO"] || 'github'
```

Comment:
maybe something like `git_host` ? or `git_provider`

Reply:
Either of those sound like a good choice.  Do you have a preference?  I think I am leaning toward git_provider.

Reply:
I like em both 👍